root.json

```
@@ -20,6 +20,7 @@
     "Gogs": "gogs.json",
     "Gollum": "gollum.json",
     "HAProxy-Letsencrypt": "haproxy-letsencrypt.json",
+    "Handbrake": "handbrake.json",
```

Minor typo. I'm unsure whether or not it would actually confuse the parsing of information, but in the doubt I'd rather have it consistent with the rock-on name from the JSON.

```suggestion
    "HandBrake": "handbrake.json",
```